taimen/config.json

```diff
@@ -61,7 +61,6 @@
         "system/lib/com.qualcomm.qti.qcril.qcrilhook@1.0.so",
         "system/lib/libdiag_system.so",
         "system/lib/libeaselcomm.so",
-        "system/lib/libeaselcontrol.amber.so",
```

Comment:
This is in vendor now (and thus would get included by default.

Reference: https://git.rip/dumps/google/taimen/-/blob/taimen-user-11-RP1A.200720.009-6720564-release-keys/all_files.txt

Reply:
Yep, that's why this line is being removed from `system-other` here.